menu_vypocty.py

```diff
@@ -0,0 +1,34 @@
+def menu():
+    print("\nZvolte jednu z možností:")
+    print("Q - Quit (Ukončit program)")
+    print("S - Square (Umocnit číslo na druhou)")
+    print("D - Double (Vynásobit číslo dvěma)")
+    print("H - Half (Vydělit číslo dvěma)")
+    print("P - Print (Zobrazit hodnotu čísla n)")
+
+# Hlavní část programu
+n = float(input("Zadejte číslo: "))
+
+while True:
+    # Zobrazit menu

```

Comment:
It is not necessary to comment on small parts of the code where it is clear what they do. It is more useful to comment on a block of code that has a specific purpose. For example, a loop that displays the menu and allows the user to make a choice.

Reply:
Good point. Instead, you can suggest to use better naming, e.g. `menu` -> `display_menu`, `choice` -> `user_action`.